.circleci/config.yml

```diff
@@ -122,6 +122,7 @@ jobs:
           key: asdf_directory-{{ .Branch }}-{{ .Revision }}
           paths:
             - /usr/local/asdf
+            - ~/.asdf
```

Comment:
I guess you can remove this commit because `~/asdf` is already cached in `asdf_directory-310988_update_libs`